src/cli/MareTF.cpp

```diff
@@ -415,6 +686,68 @@ int main(int argc, const char* const argv[]) {
 		.flag()
 		.store_into(hdriNoFilter);
 
+	bool alphaToDistance;
+	createCLI
+		.add_argument("--alpha-to-distance")
+		.help("Convert alpha channel to a distance map.")
+		.flag()
+		.store_into(alphaToDistance);
+
+	uint16_t reduce = 1;
+	createCLI
+		.add_argument("--reduce")
+		.metavar("FACTOR")
+		.help("Factor by which to downscale when building a distance map."
+			" Must be a power of 2. Overridden by --reduce-x and --reduce-y.")
+		.scan<'u', uint16_t>()
+		.default_value(reduce).store_into(reduce);
+
+	bool reduceXSet = false;
+	uint16_t reduceX = 1;
+	createCLI
+		.add_argument("--reduce-x")
+		.metavar("FACTOR")
+		.action([&](const auto&){ reduceXSet = true; })
+		.help("Factor by which to downscale width when building a distance map."
+			" Must be a power of 2.")
+		.scan<'u', uint16_t>()
+		.default_value(reduceX).store_into(reduceX);
+
+	bool reduceYSet = false;
+	uint16_t reduceY = 1;
+	createCLI
+		.add_argument("--reduce-y")
+		.metavar("FACTOR")
+		.action([&](const auto&){ reduceYSet = true; })
+		.help("Factor by which to downscale height when building a distance map."
+			" Must be a power of 2.")
+		.scan<'u', uint16_t>()
+		.default_value(reduceX).store_into(reduceY);
```

Comment:
```suggestion
		.default_value(reduceY).store_into(reduceY);
```

I do this all the time